pkg/database/database.go

```diff
@@ -0,0 +1,94 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package database
+
+import (
+	"storj.io/storj/internal/migrate"
+	dbx "storj.io/storj/pkg/database/dbx"
+	"storj.io/storj/pkg/utils"
+)
+
+// DB contains access to different database tables
+type DB interface {
```

Comment:
I don't think we need this interface, it can be a the actual implementation.

Reply:
ok